Examples/src/SharedElements.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { View, Text, Button, StyleSheet } from 'react-native';
-import { Transition, createFluidNavigator } from 'react-navigation-fluid-transitions';
+import { Transition, FluidNavigator } from 'react-navigation-fluid-transitions';
 
```

Comment:
In a transition period I'd like to have both functions available - maybe we could import both createFluidNavigator and createFluidNavigator for a little while?

Reply:
Sure, or maybe have 2 different projects for react-navigation v1 and react-navigation v2 ? Less confusing